src/Features/RemotePaymentMethods/DefaultPaymentGateways.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Gets a list of fallback methods if remote fetching is disabled.
+ */
+
+namespace Automattic\WooCommerce\Admin\Features\RemotePaymentMethods;
+
+defined( 'ABSPATH' ) || exit;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks;
+
+/**
+ * Default Payment Gateways
+ */
+class DefaultPaymentGateways {
+
+	/**
+	 * Get default specs.
+	 *
+	 * @return array Default specs.
+	 */
+	public static function get_all() {
```

Comment:
Have we given a lot of thought about how robust to make these defaults? Would it make sense to add `required_settings_keys` here as well, since I doubt they'd change all that often?

Reply:
I'm against trying to include any plugin-related logic in here.  I think this is solely the domain of plugins and we can facilitate that in their respective repos to get things started.